expression/builtin_cast_vec_test.go

```diff
@@ -70,6 +70,7 @@ var vecBuiltinCastCases = map[string][]vecExprBenchCase{
 				&timeStrGener{},
 				&dataStrGener{},
 			}},
+		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETJson}},
```

Comment:
Should we generate some jsons that can cast to valid time ?

Reply:
Added `jsonTimeGener`.